tests/cephadm/test_client.py

```diff
@@ -83,6 +83,10 @@ def setup(host):
                 "8": {
                     "4": ["rhceph-4-tools-for-rhel-8-x86_64-rpms"],
                     "5": ["rhceph-5-tools-for-rhel-8-x86_64-rpms"],
+                    "6": [
```

Comment:
As this dictionary holds live repos, don't add downstream repo here.

Reply:
As discussed, Added comment